Documentation/ApiOverview/PasswordPolicies/Index.rst

```diff
@@ -0,0 +1,146 @@
+            'validators' => [
+                \TYPO3\CMS\Core\PasswordPolicy\Validator\CorePasswordValidator::class => [
+                    'options' => [
+                        'minimumLength' => 6,
+                    ],
+                ],
+            ],
+        ],
+    ];
+
+Then assign the custom password policy `simple` to frontend and/or backend
+context:
+
+..  code-block:: php
+    :caption: config/system/settings.php | typo3conf/system/settings.php
+
+    $GLOBALS['TYPO3_CONF_VARS']['BE']['passwordPolicy'] = 'simple';
+    $GLOBALS['TYPO3_CONF_VARS']['FE']['passwordPolicy'] = 'simple';
+
+..  attention::
+    When implementing a custom password policy please refer to the
+    :ref:`secure password guidelines <security-secure-passwords>`.
+
+
+Password policy validators
+==========================
+
+TYPO3 ships with two password policy validators, which are both used in the
+default password policy.
+
+:php:`\TYPO3\CMS\Core\PasswordPolicy\Validator\CorePasswordValidator`
+---------------------------------------------------------------------
+
+This validator has the ability to ensure a complex password with a defined
+minimum length and four individual requirements.
+
+The following options are available:
+
++-----------------------------------+----------------------------+------+---------+
+| Option                            | Description                | Type | Default |
++-----------------------------------+----------------------------+------+---------+
+| :php:`minimumLength`              | Minimum length             | int  | 8       |
++-----------------------------------+----------------------------+------+---------+
+| :php:`upperCaseCharacterRequired` | Upper case character check | bool | false   |
++-----------------------------------+----------------------------+------+---------+
+| :php:`lowerCaseCharacterRequired` | Lower case character check | bool | false   |
++-----------------------------------+----------------------------+------+---------+
+| :php:`digitCharacterRequired`     | Digit check                | bool | false   |
++-----------------------------------+----------------------------+------+---------+
+| :php:`specialCharacterRequired`   | Special character check    | bool | false   |
++-----------------------------------+----------------------------+------+---------+
+
+:php:`\TYPO3\CMS\Core\PasswordPolicy\Validator\NotCurrentPasswordValidator`
+---------------------------------------------------------------------------
+
+This validator can be used to ensure, that the new user password is not
+equal to the old password. The validator must always be configured with
+the exclude action :php:`\TYPO3\CMS\Core\PasswordPolicy\PasswordPolicyAction::NEW_USER_PASSWORD`,
+because it should be excluded, when a new user account is created.
+
+
+Disable password policies globally
+==================================
+
+To disable the password policy globally (e.g. for local development) an empty
```

Comment:
Should we add a warning not to do this in production? It should be self-evident, but people are people---

Reply:
That's why I wrapped it into the application context condition to avoid it gets deployed somehow to production. In the changelog it's not done that way. But as I think of US americans (which are used to such warnings), maybe better.

Reply:
It cant do no harm ;)

Reply:
Also the changelog usually has more advanced readers then us...